ctx.go

```diff
@@ -1485,3 +1485,10 @@ func (c *Ctx) IsFromLocal() bool {
 	}
 	return c.isLocalHost(ips[0])
 }
+
+func (c *Ctx) Validate(i interface{}) error {
+	if c.app.config.Validator == nil {
+		return ErrValidatorNotRegistered
+	}
+	return c.app.config.Validator.Validate(i)
```

Comment:
We need to check here is it Ptr or Struct like https://github.com/gin-gonic/gin/blob/master/binding/default_validator.go#L55

Reply:
I'm resolving this comment because of adding default validator bloats fiber core.